packages/async-loader/async.js

```diff
@@ -1,36 +1,80 @@
-import { h, Component } from 'preact';
+// import { options, h, Component } from 'preact';
+import { h, Component, options, render } from 'preact';
 
-export default function(req) {
-	function Async() {
-		Component.call(this);
+const f = 'unmount';
```

Comment:
and maybe a more descriptive name, we are not the minifier 🙃

Reply:
done